Cargo.toml

```diff
@@ -6,12 +6,8 @@ members = ["ipa-core", "ipa-macros"]
 incremental = true
 lto = "thin"
 
-[profile.bench]
-debug-assertions = true
```

Comment:
If there's a need to have an optimized build with stricter sanity checks, then maybe we should add a dedicated config for that (something like `dev-opt`)? Or perhaps reconsider the safety / performance tradeoff for the sanity checks that are having escapes, i.e. consider converting to a regular assert rather than a debug assert.

I've actually been mostly using the release config for benchmarking -- removing this bit of config doesn't have much direct impact on what I've been doing, but it does seem to me like the right thing to do. Having to edit Cargo.toml to make the `bench` config actually produce accurate benchmark results seems confusing.

The other change I had considered making was splitting the PRSS used index tracking to a separate feature flag. I can see how that would make sense if we want to use `debug-assertions` for "small but non-negligible performance cost" and a separate flag(s) for "very significant performance cost".

Reply:
yea I think this change makes sense because we want to evaluate the performance. Just wanted to provide an explanation why it was added. It can always be enabled from command line, so I don't think it is an issue